realis_e2e_test.go

```diff
@@ -389,3 +389,26 @@ func TestRealisClient_SessionThreadSafety(t *testing.T) {
 
 	wg.Wait()
 }
+
+// Test setting and getting the quota
+func TestRealisClient_SetQuota(t *testing.T) {
+	var cpu = 2.5
+	var ram int64 = 10240
+	var disk int64 = 10240
+	resp, err := r.SetQuota("vagrant", cpu, ram, disk)
+	assert.NoError(t, err)
+	assert.Equal(t, aurora.ResponseCode_OK, resp.ResponseCode)
+
+	// Test GetQuota based on previously set values
```

Comment:
Excellent patch! Thanks for taking the time to open up a pull request.

The only change I'd make is to wrap the GetQuota test in a closure so that the testing framework shows it as a different test. making it easier to tell which test failed:

```
t.Run("TestRealisClient_GetQuota", func(t *testing.T) {
// Lines 403-413 in here
})
```

Other than that looks good to me!

Reply:
:+1: